ovs/extensions/db/arakoon/ArakoonInstaller.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, cluster_id, plugins=None):
         self.cluster_id = cluster_id
         self._dir = ArakoonClusterConfig.ARAKOON_CONFIG_DIR.format(self.cluster_id)
         self.filename = ArakoonClusterConfig.ARAKOON_CONFIG_FILE.format(self.cluster_id)
+        self._extra_globals = {'__tainted_tlog_entries_per_file': 5000}
```

Comment:
Will this only be used for new config clusters and never for already existing ones?